src/rules/maxLineLengthRule.ts

```diff
@@ -57,10 +66,11 @@ export class Rule extends Lint.Rules.AbstractRule {
                     {
                         type: "object",
                         properties: {
-                            limit: { type: "number" },
-                            "ignore-pattern": { type: "string" },
+                            [OPTION_LIMIT]: { type: "number" },
+                            [OPTION_IGNORE_PATTERN]: { type: "string" },
+                            [OPTION_CHECK_STRINGS]: { type: "boolean" },
+                            [OPTION_CHECK_REGEX]: { type: "boolean" },
                         },
-                        additionalProperties: false,
```

Comment:
Could you please put the `additionalProperties: false` back in? It's an intentional part of the schema.